trimmer_decorator.rb

```diff
@@ -0,0 +1,7 @@
+require_relative './base_decorator'
+class TrimDecorator < BaseDecorator
+  def correct_name
+    @nameablestore.correct_name[0..10] if
```

Comment:
- [ ] Please make the trimmer decorator trim the name to make it has a maximum of 10 characters because now it's trimming 11 characters It's supposed to be like that:
```ruby
@nameablestore.correct_name[0..9]
```

Reply:
well resolved! ✔️